core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/storage/ScanSourceFileResolver.scala

```diff
@@ -0,0 +1,37 @@
+package edu.uci.ics.texera.workflow.common.storage
+
+import java.nio.file.{Files, Path, Paths}
+import edu.uci.ics.amber.engine.common.storage.DatasetFileDocument
+import scala.util.{Failure, Success, Try}
+
+object ScanSourceFileResolver {
```

Comment:
Remove scan and source from name. It is a data layer thing.  Scan is one of the applications. We may reuse the logic for result as well

Reply:
renamed